### PR TITLE
Feature/payments enterprise

### PR DESCRIPTION
**Pull request by TatianaLopaeva**

### CODE REVIEW

File: web/ASC.Web.Client/src/components/pages/PaymentsEnterprise/sub-components/buttonContainer.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  );
+  useEffect(() => {
+    changeLanguage(i18n);
+  }, []);
+
+  const { t } = useTranslation("translation", { i18n });
+  return (
+    <StyledButtonContainer>
+      <Button
+        className="button"
```
Comment (AlexeySafronov) [inline web/ASC.Web.Client/src/components/pages/PaymentsEnterprise/sub-components/buttonContainer.js:79]: Is it realy necessary to style default Button component?

File: web/ASC.Web.Client/src/components/pages/PaymentsEnterprise/sub-components/headerContainer.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    changeLanguage(i18n);
+    const moment = require("moment");
+    require("moment/min/locales.min");
+    culture && moment.locale(culture);
+  }, [culture]);
+
+  const { t } = useTranslation("translation", { i18n });
+  const currentUserDate = moment().utcOffset(utcHoursOffset);
+
+  return new Date(currentUserDate).setHours(0, 0, 0, 0) <
```
Comment (AlexeySafronov) [inline web/ASC.Web.Client/src/components/pages/PaymentsEnterprise/sub-components/headerContainer.js:57]: Re-write to variables condition

File: web/ASC.Web.Client/src/store/payments/reducer.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  currentLicense: {
+    expiresDate: new Date(),
+    trialMode: false,
+  },
+};
+
+const paymentsReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case SET_SETTINGS_PAYMENTS_ENTERPRISE:
+      return Object.assign({}, state, {
```
Comment (AlexeySafronov) [inline web/ASC.Web.Client/src/store/payments/reducer.js:19]: Need to simplify

File: web/ASC.Web.Client/src/components/pages/PaymentsEnterprise/sub-components/buttonContainer.js (hunk 1 above)
Comment (TatianaLopaeva) [reply web/ASC.Web.Client/src/components/pages/PaymentsEnterprise/sub-components/buttonContainer.js]: No, it isn't. I deleted unnecessary properties and className

File: web/ASC.Web.Client/src/store/payments/reducer.js (hunk 3 above)
Comment (TatianaLopaeva) [reply web/ASC.Web.Client/src/store/payments/reducer.js:18]: I simplified it

File: web/ASC.Web.Client/src/components/pages/PaymentsEnterprise/sub-components/headerContainer.js (hunk 2 above)
Comment (TatianaLopaeva) [reply web/ASC.Web.Client/src/components/pages/PaymentsEnterprise/sub-components/headerContainer.js]: I Re-wrote  to variables condition

**Review by AlexeySafronov — APPROVED** [review]

**Pull request merged**